tensorflow/lite/toco/tflite/op_version.cc

```diff
@@ -121,6 +121,7 @@ std::string GetMinimumRuntimeVersionForModel(const Model& model) {
           {{OperatorType::kTranspose, 1}, "1.6.0"},
           {{OperatorType::kTranspose, 2}, "1.14.0"},
           {{OperatorType::kTranspose, 3}, "1.15.0"},
+          {{OperatorType::kTranspose, 5}, kPendingReleaseOpVersion},
```

Comment:
this place confused me - should I add version 4 with kPendingReleaseOpVersion ?

Reply:
Seems it's "2.3.0"?

+jdduke@